spec/models/user_spec.rb

```diff
@@ -1,16 +1,16 @@
-require 'spec_helper'
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.